Backends/CLX/fonts.lisp

```diff
@@ -72,13 +72,10 @@
          (find-font ()
            (multiple-value-bind (family face size)
                (text-style-components text-style)
-
-             (setf face   (or face :roman)
-                   family (or family :fix)
-                   size   (max 2 (climb:normalize-font-size size)))
-
+             (setf size (max 2 (climb:normalize-font-size size)))
              (let ((display (clim-clx::clx-port-display port)))
                (find-and-make-xlib-face display family face size)))))
+      (setf text-style (merge-text-styles text-style *default-text-style*))
```

Comment:
How expensive is this? Should we only do it if one of the components is unspecific?

Reply:
the answer is: not very expensive. If we want to change that (with looking if the component is not specified), I'd go with

```
           (setf face   (or face (text-style-face *default-text-style*))
                   family (or family (text-style-family *default-text-style*))
                   size   (climb:normalize-font-size size))
```

instead of merge style.

Reply:
as a side note: cleaner solution would be having text-style with wildcard components having a separate class. but this seems to be a broad change and I'm not very interested in making it now.

Reply:
I've fixed it (also I've fixed unreported issue when *default-text-style* size was numeric value.